device/mock_device/src/lib.rs

```diff
@@ -20,8 +20,8 @@ pub mod mock_data;
 pub mod mock_device_controller;
 pub mod mock_device_ffi;
 pub mod mock_device_processor;
+pub mod mock_server;
 pub mod mock_web_socket_server;
-pub mod utils;
-
 #[cfg(test)]
 pub(crate) mod test_utils;
+pub mod utils;
```

Comment:
Why has this moved? The functions in here are needed to start the extension.

Reply:
accidental during merge will fix